evap/evaluation/models.py

```diff
@@ -222,6 +236,18 @@ def __lt__(self, other):
     def __gt__(self, other):
         return (self.type, self.order, self.pk) > (other.type, other.order, other.pk)
 
+    @transaction.atomic()
+    def set_active_dropout(self):
+        # TODO@Felix: test this method
+        if self.type != Questionnaire.Type.DROPOUT:
+            raise ValueError("Can only set DROPOUT-type Questionnaires as active dropout questionnaire.")
```

Comment:
probably fair to just use an assertion -- or do we expect this to ever happen for a correct program execution?